libraries/AP_RangeFinder/AP_RangeFinder_BLPing.cpp

```diff
@@ -0,0 +1,286 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <AP_HAL/AP_HAL.h>
+#include "AP_RangeFinder_BLPing.h"
+#include <AP_SerialManager/AP_SerialManager.h>
+#include <ctype.h>
+#include <AP_HAL/utility/sparse-endian.h>
```

Comment:
Did you end up using anything from this header?

Reply:
No I didn't so removed that include, txs